packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -109,7 +109,8 @@ function computeSelectionCellKeys(
 
     for (let c = minColIdx; c <= maxColIdx; c++) {
         for (let r = minRowIdx; r <= maxRowIdx; r++) {
-            selectionCells.push(genCellKey(c, r));
+            const fieldKey = editorModel.columnMap.get(editorModel.orderedColumns.get(c)).fieldKey;
```

Comment:
If this PR gets merged with a lot of these usages, yes, I think it will make sense, but in general I want to get us away from using column indices. However, it may make sense to defer that work, and if I do, I'll add the helper.

Reply:
I went ahead and added `getFieldKeyByIndex` and `getColumnByIndex` util methods to `EditorModel`